sg: "ci open" panics

`HEAD=c10eab4e5bd20d28457b7bfb5baa6fa504ad7a4c`

```
/work/sourcegraph  main-dry-run/re-enable-yarn-cache $ sg ci open
⚠️ update check: 'git rev-list 794d7a02e4397ab4c75976ed1581a3eb10dfc1a3..origin/main -- ./dev/sg' failed: fatal: Invalid revision range 794d7a02e4397ab4c75976ed1581a3eb10dfc1a3..origin/main: exit status 128
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.glob..func11({0x140008417f0?, 0x1042d6a6c?}, {0x10722a880, 0x0, 0x140008417e8?})
	github.com/sourcegraph/sourcegraph/dev/sg/sg_ci.go:541 +0xf0
main.execAdapter.func1(0x140001b2f80)
	github.com/sourcegraph/sourcegraph/dev/sg/ffcli.go:14 +0x90
main.addAnalyticsHooks.func1(0x140001b2f80)
	github.com/sourcegraph/sourcegraph/dev/sg/analytics.go:45 +0x110
github.com/urfave/cli/v2.(*Command).Run(0x1071bc1c0, 0x140001b2ec0)
	github.com/urfave/cli/v2@v2.4.0/command.go:163 +0x50c
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0x140004c3ba0, 0x140003a7cc0)
	github.com/urfave/cli/v2@v2.4.0/app.go:434 +0x9b0
github.com/urfave/cli/v2.(*Command).startApp(0x1071bbb00, 0x140003a7cc0)
	github.com/urfave/cli/v2@v2.4.0/command.go:280 +0x7ac
github.com/urfave/cli/v2.(*Command).Run(0x140003a7cc0?, 0x1071b2620?)
	github.com/urfave/cli/v2@v2.4.0/command.go:94 +0x84
github.com/urfave/cli/v2.(*App).RunContext(0x1071bece0, {0x1066b3d70?, 0x14000126008}, {0x1400011c090, 0x3, 0x3})
	github.com/urfave/cli/v2@v2.4.0/app.go:313 +0x888
main.main()
	github.com/sourcegraph/sourcegraph/dev/sg/main.go:26 +0xb8
```

--- a/dev/sg/sg_ci.go
+++ b/dev/sg/sg_ci.go
@@ -538,7 +538,8 @@ From there, you can start exploring logs with the Grafana explore panel.
 		Description: "Open Sourcegraph's Buildkite page in browser. Optionally specify the pipeline you want to open.",
 		Action: execAdapter(func(ctx context.Context, args []string) error {
 			buildkiteURL := fmt.Sprintf("https://buildkite.com/%s", bk.BuildkiteOrg)
-			if pipeline := args[0]; pipeline != "" {
+			if len(args) > 0 && args[0] != "" {
+				pipeline := args[0]
 				buildkiteURL += fmt.Sprintf("/%s", pipeline)
 			}
 			return open.URL(buildkiteURL)

